locustempus/templates/main/course_detail.html

```diff
@@ -64,6 +65,10 @@
 {% endif %}
 
 <div class="row dashboard-container" data-cy="project-list">
+    <p>Authors: {{ course | course_faculty_string }}</p>
```

Comment:
This looks quite wonky from a UI perspective, and disrupts the card rendering in the flex box. I'm hesitant to merge this as is...maybe hide this information for the moment?